services/src/main/java/org/keycloak/protocol/oid4vc/issuance/mappers/OID4VCUserAttributeMapper.java

```diff
@@ -58,12 +47,9 @@ public class OID4VCUserAttributeMapper extends OID4VCMapper {
 
         ProviderConfigProperty userAttributeConfig = new ProviderConfigProperty();
         userAttributeConfig.setName(USER_ATTRIBUTE_KEY);
-        userAttributeConfig.setLabel("User attribute");
-        userAttributeConfig.setHelpText("The user attribute to be added to the credential subject.");
-        userAttributeConfig.setType(ProviderConfigProperty.LIST_TYPE);
-        userAttributeConfig.setOptions(
-                List.of(UserModel.USERNAME, UserModel.LOCALE, UserModel.FIRST_NAME, UserModel.LAST_NAME,
-                        UserModel.DISABLED_REASON, UserModel.EMAIL, UserModel.EMAIL_VERIFIED));
+        userAttributeConfig.setLabel("User property");
+        userAttributeConfig.setHelpText("The user attributes to be added to the credential subject.");
```

Comment:
It seems you meant "User attribute" here.
```suggestion
        userAttributeConfig.setLabel("User attribute");
        userAttributeConfig.setHelpText("The user attribute to be added to the credential subject.");
```